src/cli.ts

```diff
@@ -74,7 +74,7 @@ const mainCommand = defineCommand({
     });
 
     const _from = r.name || r.url;
-    const _to = relative(process.cwd(), r.dir);
+    const _to = process.cwd() === r.dir ? "./" : relative(process.cwd(), r.dir);
```

Comment:
Refactor: Can you verify if we could instead use this:

```suggestion
    const _to = relative(process.cwd(), r.dir) || "./";
```